moonbeam-types-bundle/index.ts

```diff
@@ -350,7 +350,7 @@ export const moonbeamDefinitions = {
         AccountId32: "H256",
         AccountInfo: "AccountInfoWithTripleRefCount",
         Address: "AccountId",
-        AuthorId: "H256",
```

Comment:
Why these changes? I'm fine with it if you like it, but to me this makes it seem like AuthorIds could also be used as AccoutIds. I realize this is how PolkadotJs generally does it too, so I'm fine if you like ti this way.

Reply:
Hum... I thought that it would allow changing all occurrences of AccountId32 more easily... Should I revert it?